### ISSUE TITLE
Format off for Community Page

### ISSUE THREAD

**Issue by swzCuroverse** [common-workflow-language/cwl-website#126]
The meeting details on the new community page are not formatted by like the rest of the page. 
* https://www.commonwl.org/community/

In general: Perhaps we don't need this on the community page but a link to a new page since it swallows up the rest of the resources of which the page was intended.  We already have a mention of the weekly chat in the bullet points, so it is extra confusing.  Perhaps that should link to a nicely formatted page about the meetings if people are interested.  

To look like the rest of the page/general guidelines:
(1) Follow look and feel of rest of site
* Titles should be in all caps --> Agenda and Notes not Agenda and notes (and/if/but not caps)
* If you have a list of items --> Use the bullets
* We use numbers no where else on the site 
*  Periods only needed at end of sentences ([](https://www.commonwl.org/community/#agenda-and-notes)https://w3id.org/cwl/meeting_minutes.) 
*  Hyperlinks should be given context ([](https://www.commonwl.org/community/#agenda-and-notes)https://w3id.org/cwl/meeting_minutes.) and alt text instead of the direct link. 
* Avoid long hyperlinks if possible:  [EMEA](https://en.wikipedia.org/wiki/Europe,_the_Middle_East_and_Africa)-Americas: weekly [every Wednesday at 16:00 UTC from March-November and 17:00 UTC from November-March](https://arewemeetingyet.com/New%20York/2021-07-21/12:00/w/Weekly%20CWL%20Video%20Chat#eyJ1cmwiOiJodHRwczovL21lZXQuaml0LnNpL0NvbXBldGVudEJyYW5jaGVzTm9taW5hdGVXZWFrbHkifQ==)
* You should try to not have have section headers just start mid-page, for a good example to do this correctly see here:  https://www.commonwl.org/features
*  Balance new sections with existing format and content: Code of conduct already lives here: https://www.commonwl.org/code-of-conduct/ -- link to it perhaps in the list of bullet points above instead of having a new section for it.  
*  Consider making a new page when a topic seems to overwhelm the content of the page and/or is not of interest to all participants or very detailed on one topic
* Avoid hyperlinks when you can just put the detail easily on the page -- aka EMEA (Europe, the Middle East and Africa) is way more informative than linking to a wikipedia article.  Same with APAC.

(2) Make things are new person/user friendly as possible: No explanation made about what the Weekly Meeting is, why one would attend or any context.  Again, lots of work to make things as user-friendly in mind so we should keep doing this.  

In general, we should have a review process since a bunch of work was made to make the website look good , organized, be approachable and consistent.  Otherwise, I fear we will lose the headway we made quickly with the reformat.

**Comment by lunacodes** [comment]
A few thoughts on the page, now that I'm looking at it again:

1. I think some of the image sizing & padding still needs a bit of work (I can take care of that)
2. Images need to have Alt Text (this should be added to the general Guidelines), for accessibility. 
3. The "Agenda and notes" and "When" seem bit overkill as section headings, given the minimal content. Were they intended semantically to be headings? (If not, then the font size should be adjusted via css class. I can write instructions for that, as well as create some preset sizes, if that would be helpful).

**Comment by swzCuroverse** [comment]
Hi Luna - I would suggest the following - make a new page for the Weekly Meetings and adjust that and have the text go back to the old text and adjusting the following text for the last bullet point:

* Please consider joining one of the weekly CWL Community Video Chats for discussion about community topics such as upcoming features, outstanding bugs, new CWL implementations or projects and community building activities. 

--->  where the link is the details for the meeting, times, login as less as the previously in the bullet google groups sign in.

**Comment by swzCuroverse** [comment]
Also started the document for Guidelines here:
https://docs.google.com/document/d/1uh18NImnHWCzibpluVpZ7lY2xmwk_VaSxD2-YyXzBQA/edit?usp=sharing

**Comment by lunacodes** [comment]
> would suggest the following - make a new page for the Weekly Meetings and adjust that and have the text go back to the old text and adjusting the following text for the last bullet point:

@swzCuroverse Makes sense. I have a PR #131 with revised formatting already - so I'll add this there. I can name the new page `meetings.md` and put it as a dropdown under content, if that makes sense.

Just to clarify:

* Community: Bullet Points + Code of Conduct
* New page: CWL Online Meetings + Using Matrix

Is that correct?

**Comment by swzCuroverse** [comment]
Yes, thank you.
